src/plone/restapi/tests/test_services_mail.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+from Products.MailHost.interfaces import IMailHost
+
+from plone.app.testing import setRoles
+from plone.app.testing import SITE_OWNER_NAME
+from plone.app.testing import SITE_OWNER_PASSWORD
+from plone.app.testing import TEST_USER_ID
+from plone.restapi.testing import PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
+from plone.restapi.testing import RelativeSession
+from zope.component import getUtility
+from plone.registry.interfaces import IRegistry
+
+import transaction
+import unittest
+
+try:
+    from Products.CMFPlone.factory import _IMREALLYPLONE5  # noqa
+except ImportError:
+    PLONE5 = False
+else:
+    PLONE5 = True
+
+
+@unittest.skipIf(not PLONE5, 'Just Plone 5 currently.')
+class TestUsersEndpoint(unittest.TestCase):
```

Comment:
@sneridagh TestUsersEndpoint? Pls rename to EmailNotificationEndpoint

Reply:
Done.